### PR TITLE
New package: tiramisu-2.0alpha

### PR DESCRIPTION
**Pull request by DanKGooGLy**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/tiramisu/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'tiramisu'
+pkgname=tiramisu
+version=2.0alpha
```
Comment (paper42) [inline srcpkgs/tiramisu/template:3]: I think we should wait for 2.0 or package 1.0

File: srcpkgs/tiramisu/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+revision=1
+build_style=gnu-makefile
+hostmakedepends="libglib-devel pkg-config"
+short_desc="Desktop notifications, the UNIX way"
+maintainer="DanKGooGLy <dankgoogly@riseup.net>"
+license="MIT"
+homepage="https://github.com/Sweets/tiramisu"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
```
Comment (paper42) [inline srcpkgs/tiramisu/template:11]: I would remove ${homepage} from distfiles

**Comment by Johnnynator** [comment]
The pull bot active on your fork managed to drop your commit in this PR and github marked it ad merged since it had the same HEAD as master.

**Comment by DanKGooGLy** [comment]
Oh.

**Comment by DanKGooGLy** [comment]
What do I do now? Reopen the PR?

**Comment by Johnnynator** [comment]
I think github doesn't allow to reopen a merged PR, so I would recommend just to make a new one (and from a different branch).

**Comment by DanKGooGLy** [comment]
Alright.

**Pull request merged**